app.rb

```diff
@@ -16,26 +16,82 @@ def initialize
   end
 
   def list_people
-    ...
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
   end
 
   def list_books
-    ...
+    @books.each { |book| puts "Title: \"#{book.title}\" , Author: #{book.author}"}
+  end
+
+  def create_teacher
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Specialization:'
+    specialization = gets.chomp
+    new_person = Teacher.new(age, specialization, name)
+    @people << new_person
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Has parent permission? [Y/N]:'
+    permission = gets.chomp
+    if permission.upcase == 'Y'
+      permission = true
+    else
+      permission = false
+    end
+    new_person = Student.new(age, '', name, permission)
+    @people << new_person
+    puts 'Person created successfully'
```

Comment:
- [ ] Good job so far 👍 When I try to create a student in your console app, I get the following error. Kindly rectify this.
- Error
![Annotation 2022-05-19 043628](https://user-images.githubusercontent.com/85113567/169199640-a99d7b37-b391-451f-83a7-04ad3f33dbb9.jpg)